### DISCUSSION TITLE
instancenorm with default affine=True?

### DISCUSSION THREAD

**Discussion by wyli** [Project-MONAI/MONAI#2091] [Ideas]
should the instancenorm's default `affine` parameter be `True`, so that the behaviour is more consistent with the batch norm?

cc @myron

**Comment by myron** [comment]
yes, we should.
 It defaults to True for Batchnorm and Groupnorm in PT. I'm not sure why it's false for instancenorm.
